aiven/client/speller.py

```diff
@@ -4,21 +4,24 @@
 # MIT license: www.opensource.org/licenses/mit-license.php
 
 
-def suggest(word_to_check, known_words):
-    def get_correction(word):
+from typing import Iterable, Optional
+
+
+def suggest(word_to_check: str, known_words: Iterable[str]) -> Optional[str]:
+    def get_correction(word: str) -> Optional[str] :
         """Most probable spelling correction for word."""
         candidates = get_candidates(word)
         return next(iter(candidates)) if candidates else None
 
-    def get_candidates(word):
+    def get_candidates(word: str) -> Iterable[str]:
```

Comment:
Probably can be `Iterator[str]` instead. The result is only used once in `get_correction()`